.eslintrc.json

```diff
@@ -0,0 +1,32 @@
+{
+  "extends": "jquery",
```

Comment:
From your question below, maybe: `extends: lib/wet-boew/.eslintrc.json`
Other large projects sometimes just create a separate ESlint rules repo that can be referenced. EX https://github.com/jquery/eslint-config-jquery

Reply:
Ok. I'll modify this PR to go in that direction. Btw that will result in this file's removal, since I initially copied it from wet-boew/wet-boew (without any changes).